daemon/main.cpp

```diff
@@ -47,7 +47,11 @@ FILE *dmLog = nullptr;
 
 bool AlreadyRunning()
 {
+#ifdef ANDROID
+    const std::string pidFile = "/data/hdcp/hdcpd.pid";
```

Comment:
Could you please #define a constant string which changes for Android instead of using an #ifdef in the code?